src/iterative_ensemble_smoother/_iterative_ensemble_smoother.py

```diff
@@ -88,34 +88,34 @@ def update_step(
         :param inversion: The type of subspace inversion used in the algorithm, defaults
             to exact.
         """
-        S = sensitivity_matrix
-        A = centered_anomaly_matrix
+        Y = response_ensemble
```

Comment:
This looks a bit strange, as it is just essentially a renaming of input parameters, but I understand it then connects better to the mathematics in the code below

Reply:
I agree that it looks a bit strange, so I've pushed a commit proposing to use longer names instead.
I've still kept the more "mathy"-names in the C++ code though.
Perhaps that's not a bad way to go about it...
Do you find this a bit easier to read @pinkwah ?